modules/network-observability-operator-uninstall.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+
+// * installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-operator-uninstall_{context}"]
+
+= Uninstalling the Network Observability operator
+You can install the Network Observability operator using the {product-title} web console Operator Hub, working in the *Operators* -> *Installed Operators* area.
+
+.Procedure
+
+. Uninstall the `FlowCollector` custom resource.
+.. Click the `FlowCollector`  next to the  *Network Observability Operator* in the *Provided APIs* column. 
```

Comment:
I think you can omit the article before `FlowCollector`. There is an extra space after it too.